spec/requests/works_spec.rb

```diff
@@ -127,7 +132,7 @@
         expect(response).to have_http_status(:found)
         work = Work.last
         expect(work.contributors.size).to eq 2
-        expect(work.attached_files.size).to eq 2
+        expect(work.attached_files.size).to eq 1
```

Comment:
I'd like to show that we ignore destroyed files and can add more than one.

Reply:
I'll work on that now.

Reply:
done.